packages/menu-button/src/index.js

```diff
@@ -78,12 +78,24 @@ let Menu = ({ children }) => (
     didUpdate={manageFocusOnUpdate}
     getSnapshotBeforeUpdate={checkIfAppManagedFocus}
   >
-    {context => <Provider value={context}>{children}</Provider>}
+    {context => (
+      <Provider value={context}>
+        <Consumer>
```

Comment:
Should be able to skip the consumer and use `context.state.isOpen` yeah?

```jsx
<Provider value={context>
  {typeof children === 'function' ? children({ isOpen: context.state.isOpen }) : children }
</Provider>
```

Reply:
Done.